packages/carousel/src/css/index.js

```diff
@@ -89,6 +72,32 @@ export default {
     '&[hidden]': { transform: 'translate(50%, -50%) scale(0)' }
   },
 
+  '.psds-carousel__controls__control__button': {
+    ...resetButton,
+    alignItems: 'center',
+    borderRadius: '100%',
+    boxShadow: '0px 4px 14px rgba(0, 0, 0, 0.25)',
+    display: 'flex',
+    height: '36px',
+    justifyContent: 'center',
+    width: '36px',
+
+    '&:focus': { outline: 'none' },
+    '&:hover': { cursor: 'pointer' }
+  },
+  [`.psds-carousel__controls__control__button.psds-theme--${themeNames.dark}`]: {
+    color: colorsTextIcon.highOnLight,
+    background: 'rgba(255, 255, 255, 0.6)',
```

Comment:
some special button colors are used here. @sweeting and I worked together to nail these down